rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/schedule/ScheduleAdminController.java

```diff
@@ -379,4 +386,43 @@ public ResponseEntity<?> scheduleSourceUploadForReleaseComponents()throws TExcep
         HttpStatus status = HttpStatus.ACCEPTED;
         return new ResponseEntity<>(requestStatus, status);
     }
+
+    @PostMapping(SCHEDULE_URL + "/status")
+    public ResponseEntity<Map<String, Object>> checkServiceStatus(@RequestBody Map<String, String> requestBody) {
+        try {
+            User sw360User = restControllerHelper.getSw360UserFromAuthentication();
+
+            String serviceName = requestBody.get("serviceName");
+            if (serviceName == null || serviceName.isBlank()) {
+                return ResponseEntity.badRequest()
+                        .body(Map.of("error", "Service name is required"));
+            }
+
+            RequestStatus requestStatus = scheduleService.isServiceScheduled(serviceName, sw360User);
+
+            Map<String, Object> response = Map.of(
+                    "serviceName", serviceName,
+                    "isScheduled", requestStatus == RequestStatus.SUCCESS
+            );
+
+            return ResponseEntity.ok(response);
+        } catch (TException e) {
+            return ResponseEntity.status(HttpStatus.INTERNAL_SERVER_ERROR)
+                    .body(Map.of("error", "Failed to check service status"));
+        }
+    }
+
+    @GetMapping(SCHEDULE_URL + "/isAnyServiceScheduled")
+    public ResponseEntity<Boolean> isAnyServiceScheduled() {
+        try {
+            User sw360User = restControllerHelper.getSw360UserFromAuthentication();
+            boolean isAnyServiceScheduled = scheduleService.isAnyServiceScheduled(sw360User) == RequestStatus.SUCCESS;
+
+            return ResponseEntity.ok(isAnyServiceScheduled);
+        } catch (AccessDeniedException e) {
+            return ResponseEntity.status(HttpStatus.FORBIDDEN).build();
+        } catch (TException e) {
+            return ResponseEntity.status(HttpStatus.INTERNAL_SERVER_ERROR).build();
+        }
```

Comment:
No need to capture these exceptions, they are handled by `RestExceptionHandler`.